src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -864,6 +885,25 @@ def pin_subvolume_group(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def check_subvolume_group(self, **kwargs):
+        volname = kwargs['vol_name']
+        ret = None
+        volume_exists = False
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                volume_exists = True
+                res = checkdir(fs_handle, self.volspec.base_dir, filter_entries=[
+                               Trash.GROUP_NAME.encode('utf-8'), "_nogroup".encode('utf-8')])
```

Comment:
Shouldn't be filtering `_nogroup` right ? @nmshelke What did we decide on `_nogroup` handling with CSI team ?

Reply:
Also needs to filter other internal directories like _index etc.

Reply:
@kotreshhr We should filter _nogroup, _index, _legacy, _deleting directories.